hydradx/model/indexer_utils.py

```diff
@@ -390,6 +397,152 @@ def get_fee_pcts(data, asset_id):
     return fee_pcts
 
 
+def get_current_stableswap_pools(asset_info: dict[int: AssetInfo] = None):
+    stableswap_data = {
+        pool: get_latest_stableswap_data(pool)
+        for pool in get_stablepool_ids()
+    }
+    if asset_info is None:
+        asset_info = get_asset_info_by_ids()
+    stableswap_pools = []
+    for pool in stableswap_data.values():
+        if min(pool['liquidity'].values()) > 0:
+            stableswap_pools.append(
+                StableSwapPoolState(
+                    tokens={asset_info[tkn_id].symbol: pool['liquidity'][tkn_id] for tkn_id in pool['liquidity']},
+                    amplification=pool['finalAmplification'],
+                    trade_fee=pool['fee'],
+                    unique_id=asset_info[pool['pool_id']].name,
+                )
+            )
+    return stableswap_pools
+
+def get_current_omnipool():
+    asset_info = get_asset_info_by_ids()
+    for asset in asset_info.values():
+        if asset.asset_type == 'StableSwap':
+            asset.symbol = asset.name
+    current_block = get_current_block_height()
+    omnipool_data = get_omnipool_asset_data(min_block_id=current_block - 10000, max_block_id=current_block)
+    liquidity = {}
+    lrna = {}
+    for item in reversed(omnipool_data):
+        if asset_info[item['assetId']].symbol not in liquidity:
+            liquidity[asset_info[item['assetId']].symbol] = int(item['balances']['free']) / (10 ** asset_info[item['assetId']].decimals)
+            lrna[asset_info[item['assetId']].symbol] = int(item['assetState']['hubReserve']) / (10 ** asset_info[1].decimals)
+        if len(liquidity) == len(asset_info):
```

Comment:
It looks like you think asset_info will only have omnipool assets, but in reality it should have many more assets than are in Omnipool, right?